services/exposurelog/values.yaml

```diff
@@ -1,6 +1,8 @@
 # Default values for exposurelog.
```

Comment:
Needs helm-docs comments added for all the settings.

Reply:
Done